### ISSUE TITLE
Figaro: key not found: "UPLOADCARE_PUBLIC_KEY"

### ISSUE THREAD

**Issue by georgesve** [uploadcare/uploadcare-ruby#117]
### key not found: "UPLOADCARE_PUBLIC_KEY"
When using Figaro for storing keys in ``application.yml`` I get the following error message when running ``rails c`` or ``rails s``: 

```
/Users/xx/.rvm/gems/ruby-3.1.2/gems/uploadcare-ruby-4.2.0/lib/uploadcare.rb:48:in `fetch': key not found: "UPLOADCARE_PUBLIC_KEY" (KeyError)

from /Users/xx/.rvm/gems/ruby-3.1.2/gems/uploadcare-ruby-4.2.0/lib/uploadcare.rb:48:in `<module:Uploadcare>'
 ```

It works when I manually import the keys like so 
```
export UPLOADCARE_PUBLIC_KEY=your_public_key
export UPLOADCARE_SECRET_KEY=your_private_key
```
But I get the following warning + I have to manually export keys on every new terminal I work on:
````
WARNING: Skipping key "UPLOADCARE_PUBLIC_KEY". Already set in ENV.
WARNING: Skipping key "UPLOADCARE_SECRET_KEY". Already set in ENV.
```` 

#### uploadcare.rb
```
Uploadcare::Rails.configure do |config|
  config.public_key = ENV.fetch('UPLOADCARE_PUBLIC_KEY', 'demopublickey')
  config.secret_key = ENV.fetch('UPLOADCARE_SECRET_KEY', 'demosecretkey')
 ...
end
```
### Expected behavior

I should be able to set the variables in application.yml

#### gemfile.lock
````
uploadcare-rails (3.2.0)
      rails (>= 4)
      uploadcare-ruby (>= 4.2)
uploadcare-ruby (4.2.0)
      api_struct (~> 1.0.1)
      mimemagic
      parallel
      retries
dry-configurable (= 0.13.0)
````

**Comment by kraft001** [comment]
@georgesve Yes, indeed, ENV keys were fetched in the gem before Figaro initialized them.
Fixed in the new version 4.3.1, you can find a sample application with Figjam [here](https://github.com/uploadcare/uploadcare-rails-example/tree/with_figaro). I used Figjam as a new version of Figaro, as it's described in [this comment](https://github.com/laserlemon/figaro/issues/290#issuecomment-1310934776)
Please, try it and let me know if you still experience the issue.

**Comment by georgesve** [comment]
@kraft001 problem solved with your fix ! thanks 👍
